.github/workflows/cd-aws.yml

```diff
@@ -134,6 +134,13 @@ jobs:
         run: |
           curl -sSL -o /usr/local/bin/argocd https://github.com/argoproj/argo-cd/releases/latest/download/argocd-linux-amd64
           chmod +x /usr/local/bin/argocd
+      - name: delete job-migrate
```

Comment:
I think that it is too aggressive, and it shouldn't be part of GH deployment, it should be managed from Argo

Reply:
I'll remove it